scripts/templates/docker-compose.template

```diff
@@ -36,6 +36,7 @@ hydroshare:
     POSTGIS_USER: postgres
     PGPASSWORD: postgres
     SOLR_HOST: solr
+    TMP: /hs_tmp
```

Comment:
@aphelionz This TMP environment variable needs to be added into scripts/templates/docker-compose-local-irods.template as well so that when USE_LOCAL_IRODS setting is true, TMP environment variable is copied over to docker-compose.yml as well.